oproldan1/mktg-webdev-challenge
# Challenge solutions

@mwickett @kylemac @alexcarpenter @ashleemboyer @brkalow

### Description

This Pull Request is for the HashiCorp take home marketing web dev challenge. It is a coded solution for all the challenge prompts.

### Task

The original repository for this challenge can be found here: https://github.com/hashicorp/mktg-webdev-challenge

### Testing Instructions

Clone this repository to your local machine, open in IDE, and run `npm install`. After installation is complete, run `npm start` in your terminal. Go to localhost:3000/home for challenge instructions, and navigate to localhost:3000/people to see the challenge solutions provided.

- [ x ] I added a descriptive title to this PR.
- [ x ] I filled out the **Description**, **Task**, and **Testing Instructions** sections above.
- [ x ] I ran `npm start` before pushing to repository to identify and fix any code issues.
- [ x ] I did a quick check to make sure my code changes follows Airbnb Style Guidelines
- [ x ] I have tagged @mwickett @kylemac @alexcarpenter @ashleemboyer, and @brkalow as reviewers for this PR.
